gui/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pyinstaller==6.2.0
+PySide6==6.5.3
+PyYAML==5.3.1
```

Comment:
Good question!
I initially tried 5.4.1, and it failed because of a weird issue with a dependency on cython:
https://github.com/yaml/pyyaml/issues/601
So I backed off to 5.3.1 and it worked.
I didn't try 6.0.1. I'll give that a shot now and see!

Reply:
I just tried 6.0.1. Works like a charm!
I did not test the application with this, but I looked at the calls in the GUI, and it's just `yaml.safe_load` and `yaml.safe_dump`, which are interfaces that haven't changed in over 5 years.
For due diligence it might be good to run the GUI with the new version of PyYAML to make double sure.

Reply:
I use safe_load and safe_dump anyway, since it's best practice to use them.

Reply:
Those `_safe` calls have paid off very well here, since the APIs for those are clearly very stable! 👍